gql/resolver.go

```diff
@@ -31,23 +33,22 @@ type dealListResolver struct {
 // resolver translates from a request for a graphql field to the data for
 // that field
 type resolver struct {
+	cfg        *config.Boost
```

Comment:
Looks like we are only using `r.cfg.LotusDealmaking.MaxDealsPerPublishMsg` -- maybe pass only that value, rather than the whole `config.Boost` ?